### PR TITLE
Experiment: running prettier on our JS and CSS files

### PR DESCRIPTION
**Pull request by smcmurtry**
# Summary | Résumé

Prettying up our JS and CSS files. This means we never have to have a conversation about how to format these files, it just happens automatically!

### CODE REVIEW

File: scripts/run_tests.sh [hunk 1]
```diff
@@ -34,8 +34,11 @@ display_result $? 1 "Code style check (flake8)"
 isort --check-only ./app ./tests
 display_result $? 2 "Import order check"
 
+npx prettier --check app/assets/javascripts app/assets/stylesheets
```
Comment (smcmurtry) [inline scripts/run_tests.sh:37]: Adding it to the `run_tests.sh` script

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Ignore minified js:
+*.min.js
+
+# Ignore all HTML files:
+*.html
+
+# Ignore minified/compiled css
+app/assets/stylesheets/index.css
```
Comment (smcmurtry) [inline .prettierignore:8]: Ignoring compiled/minified JS and CSS

File: Makefile [hunk 3]
```diff
@@ -71,4 +71,5 @@ run-dev:
 .PHONY: format
 format:
 	isort ./app ./tests
-	black ./app ./tests
\ No newline at end of file
+	black ./app ./tests
+	npx prettier --write app/assets/javascripts app/assets/stylesheets
```
Comment (smcmurtry) [inline Makefile:75]: Adding prettier to `make format`

**Review by AntoineAugusti — APPROVED** [review]
Nice

**Pull request merged**